src/Component/ExpressionInput/NumberExpressionInput/NumberExpressionInput.tsx

```diff
@@ -45,6 +46,7 @@ import './NumberExpressionInput.css';
 
 export interface NumberExpressionInputProps {
   className?: string;
+  asSlider?: boolean;
```

Comment:
I'd suggest to call the property just "slider".
```suggestion
  slider?: boolean;
```